toolkit/tools/roast/roast.go

```diff
@@ -87,10 +92,13 @@ func main() {
 		logger.Log.Panicf("Failed loading image configuration. Error: %s", err)
 	}
 
+	timestamp.Stamp.Start()
 	err = generateImageArtifacts(*workers, inDirPath, outDirPath, *releaseVersion, *imageTag, tmpDirPath, config)
 	if err != nil {
 		logger.Log.Panic(err)
 	}
+	timestamp.Stamp.RecordToCSV("generateImageArtifacts", "finishing up")
+	timestampcsvparser.OutputCSVLog(filepath.Dir(*timestampFile))
```

Comment:
I don't like the idea of the code for `roast` now having to know where all other tools have put their timestamp logs. This creates a very nasty dependency. I'd suggest each tool logging their own timestamps into a CSV and if we need anything, that spans across multiple tool runs, that should be a separate step inside out `make` build configuration, because the timestamps directories are visible and known at this higher level.

Reply:
Will address in a future PR.